keyboards/yynmt/acperience12/readme.md

```diff
@@ -0,0 +1,19 @@
+# Acperience12
+
+![Acperience12](https://raw.githubusercontent.com/yynmt/Acperience12/main/images/main_image.jpg)
+
+Acperience12 is 12 keys macropad.
+
+* Keyboard Maintainer: [yynmt](https://github.com/yynmt)
+* Hardware Supported: The A12 PCB
+* Hardware Availability: [This repository](https://github.com/yynmt/Acperience12) has the plate files and other files.
+
+Make example for this macropad (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader?

Reply:
How do I describe it?
It doesn't seem to be described in the readme.md of other keyboards.
To enter the bootloader, just press the reset button.
This is the same as the other keyboards.

Reply:
It's a newer requirement that we've added.  Basically, where the reset button is located, or if yu have bootmagic enabled, what key is it.  That sort of information.